README.md

```diff
@@ -1,15 +1,15 @@
 ![Seneca](http://senecajs.org/files/assets/seneca-logo.png)
 > A [Seneca.js][] utility plugin
 
-# seneca-basic 
+# seneca-basic
 [![Build Status][travis-badge]][travis-url]
 [![Gitter chat][gitter-badge]][gitter-url]
 
 This plugin is included with the main seneca module and provides a
 small set of basic utility action patterns.
 
 - __Version:__ 0.3.0
-- __Tested on:__ [Seneca](https://github.com/senecajs/seneca) 0.7
+- __Tested on:__ [Seneca](https://github.com/senecajs/seneca) 1.4.0
 - __Node:__ 0.10, 0.12, 4
 - __License__ [MIT][]
 
```

Comment:
please update readme using as model the https://github.com/senecajs/seneca-auth one